toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ja_JP">
+<context>
+    <name>ArrowToolOptionsBox</name>
+    <message>
+        <source>Pick:</source>
+        <translation>選択オブジェクト：</translation>
+    </message>
+    <message>
+        <source>Position</source>
+        <translation>Position</translation>
+    </message>
+    <message>
+        <source>E/W:</source>
+        <translation>E/W:</translation>
+    </message>
+    <message>
+        <source>N/S:</source>
+        <translation>N/S:</translation>
+    </message>
+    <message>
+        <source>Z:</source>
+        <translation>Z:</translation>
+    </message>
+    <message>
+        <source>SO:</source>
+        <translation>SO:</translation>
+    </message>
+    <message>
+        <source>Rotation</source>
+        <translation>Rotation</translation>
+    </message>
+    <message>
+        <source>Scale</source>
+        <translation>Maßstab</translation>
+    </message>
+    <message>
+        <source>Global:</source>
+        <translation>Global:</translation>
+    </message>
+    <message>
+        <source>H:</source>
+        <translation>H:</translation>
+    </message>
+    <message>
+        <source>V:</source>
+        <translation>V:</translation>
+    </message>
+    <message>
+        <source>Maintain:</source>
+        <translation>値を固定:</translation>
+    </message>
+    <message>
+        <source>Shear</source>
+        <translation>Scheren</translation>
+    </message>
+    <message>
+        <source>Center</source>
+        <translation type="vanished">Mitte</translation>
+    </message>
+    <message>
+        <source>Lock</source>
+        <translation type="vanished">Lock</translation>
+    </message>
+    <message>
+        <source>(</source>
+        <translation>(</translation>
+    </message>
+    <message>
+        <source>)</source>
+        <translation>)</translation>
+    </message>
+    <message>
+        <source>Center Position</source>
+        <translation>Mittlere Position</translation>
+    </message>
+</context>
+<context>
+    <name>BrushTool</name>
+    <message>
+        <source>Thickness</source>
+        <translation type="vanished">[Dicke]</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Accuracy:</source>
+        <translation>Genauigkeit:</translation>
+    </message>
+    <message>
+        <source>Selective</source>
+        <translation>Selektiv</translation>
+    </message>
+    <message>
+        <source>Preset:</source>
+        <translation>Vorlage:</translation>
+    </message>
+    <message>
+        <source>Break Sharp Angles</source>
+        <translation type="vanished">Scharfe Winkel brechen</translation>
+    </message>
+    <message>
+        <source>Pencil Mode</source>
+        <translation type="vanished">Bleistift Modus</translation>
```

Comment:
They can also be written with a hyphen. For the moment I think it is stylistically better for easier reading.

Regel 22:

Man kann einen Bindestrich in unübersichtlichen Zusammensetzungen setzen <§ 45 (2)>.

    Mehrzweck-Küchenmaschine
    Lotto-Annahmestelle
    Umsatzsteuer-Tabelle

Dies gilt auch für fremdsprachliche Fügungen aus zwei Substantiven <§ 45 E1>.

    Desktop-Publishing
    Shopping-Center

📔  Dabei sollte der Bindestrich eine Haupttrennfuge markieren.

    Flüssigwasserstoff-Tank (nicht: Flüssigwasser-Stofftank oder Flüssig-Wasserstofftank)